<?xml version="1.0"?>
<launch>

    <arg name="publisher_topic_name" default="global_location"/>
    <arg name="imu_sub_topic_name" default= "arsys/imu"/>
    <arg name="gps_sub_topic_name" default="arsys/gps"/>
    <arg name="rc_correction_topic_name" default="road_center_correction"/>
    <arg name="g2l_queue_size" default="1000"/>
    <arg name="radius_of_interest" default="50"/>
    <arg name="path_fitting_deg" default="4"/>
    <arg name="cam_link" default="cam1_link"/>
    <arg name="path_resolution" default="0.5"/>

    <node pkg="localization" type="global_localization_node" name="global_localization">
        <param name="publisher_topic_name" value="$(arg  publisher_topic_name)"/>
        <param name="imu_sub_topic_name" value= "$(arg  imu_sub_topic_name)"/>
        <param name="gps_sub_topic_name" value="$(arg  gps_sub_topic_name)"/>
        <param name="rc_correction_topic_name" value="$(arg  rc_correction_topic_name)"/>
        <param name="g2l_queue_size" value="$(arg  g2l_queue_size)"/>
        <param name="radius_of_interest" value="$(arg  radius_of_interest)"/>
        <param name="path_fitting_deg" value="$(arg  path_fitting_deg)"/>
        <param name="cam_link" value="$(arg  cam_link)"/>
        <param name="path_resolution" value="$(arg  path_resolution)"/>
    </node>

</launch>